pkgs/servers/sql/pgpool/pgpool.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/src/main/main.c b/src/main/main.c
+index eec3c3e4..d6842980 100644
+--- a/src/main/main.c
++++ b/src/main/main.c
+@@ -333,15 +333,7 @@ main(int argc, char **argv)
+ 	 */
+ 	if (strcmp("", pool_config->pool_passwd))
+ 	{
+-		char		pool_passwd[POOLMAXPATHLEN + 1];
+-		char		dirnamebuf[POOLMAXPATHLEN + 1];
+-		char	   *dirp;
+-
+-		strlcpy(dirnamebuf, conf_file, sizeof(dirnamebuf));
+-		dirp = dirname(dirnamebuf);
+-		snprintf(pool_passwd, sizeof(pool_passwd), "%s/%s",
+-				 dirp, pool_config->pool_passwd);
+-		pool_init_pool_passwd(pool_passwd, POOL_PASSWD_R);
++		pool_init_pool_passwd(pool_config->pool_passwd, POOL_PASSWD_R);
+ 	}
+ 
+ 	pool_semaphore_create(MAX_NUM_SEMAPHORES);
+diff --git a/src/watchdog/wd_lifecheck.c b/src/watchdog/wd_lifecheck.c
+index a24cad85..ae29554e 100644
+--- a/src/watchdog/wd_lifecheck.c
++++ b/src/watchdog/wd_lifecheck.c
+@@ -26,6 +26,7 @@
+ #include <ctype.h>
+ #include <time.h>
+ #include <string.h>
++#include <stdint.h>
```

Comment:
Is this still required? We have glibc 2.27 by default now

Reply:
When I was upgrading, I started without any patches and it failed to compile. Perhaps my version of nixpkgs was slightly outdated? When did that change?